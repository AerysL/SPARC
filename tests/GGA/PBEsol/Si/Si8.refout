***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:27:03 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 30 30 30
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 23
NSTATES: 30
EXCHANGE_CORRELATION: GGA_PBEsol
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 1.16E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: PBEsol/Si/Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.34 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  PBEsol/Si/Si8.out_1
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  PBEsol/Si/Si.psp8
Pseudocharge radii of atom type 1  :  7.82 7.82 7.82 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  42.85 MB
Estimated memory per processor     :  1.79 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9217798357E+00        2.370E-01        0.232
2            -3.9207850693E+00        1.782E-01        0.055
3            -3.9163345181E+00        3.613E-02        0.054
4            -3.9162851031E+00        2.104E-02        0.055
5            -3.9164429661E+00        1.170E-02        0.054
6            -3.9164586567E+00        6.056E-03        0.053
7            -3.9165019957E+00        4.338E-03        0.056
8            -3.9165034005E+00        1.159E-03        0.052
9            -3.9165039986E+00        4.869E-04        0.052
10           -3.9165040646E+00        1.696E-04        0.052
11           -3.9165040892E+00        3.313E-05        0.058
12           -3.9165040813E+00        1.921E-05        0.051
13           -3.9165040937E+00        8.307E-06        0.053
14           -3.9165040969E+00        3.467E-06        0.055
15           -3.9165040910E+00        1.851E-06        0.049
16           -3.9165040961E+00        1.099E-06        0.049
17           -3.9165040898E+00        4.985E-07        0.044
Total number of SCF: 17    

k = [0.000, 0.000, 0.000]
Occupation of state 27 = 0.000000000000000.
Occupation of state 30 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9165040845E+00 (Ha/atom)
Total free energy                  : -3.1332032676E+01 (Ha)
Band structure energy              :  1.6535972688E+00 (Ha)
Exchange correlation energy        : -9.7361556169E+00 (Ha)
Self and correction energy         : -1.0219506067E+02 (Ha)
Entropy*kb*T                       : -4.3030158451E-04 (Ha)
Fermi level                        :  2.4127533373E-01 (Ha)
RMS force                          :  1.6724415135E-02 (Ha/Bohr)
Maximum force                      :  5.1148837030E-02 (Ha/Bohr)
Time for force calculation         :  0.009 (sec)
Pressure                           :  9.5357862045E+00 (GPa)
Maximum stress                     :  9.5776637198E+00 (GPa)
Time for stress calculation        :  0.036 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1.179 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
