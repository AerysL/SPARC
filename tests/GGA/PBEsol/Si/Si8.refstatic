***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Si:
      0.0100000000       0.0200000000       0.0250000000
      0.5000000000       0.5000000000       0.0000000000
      0.0000000000       0.5000000000       0.5000000000
      0.5000000000       0.0000000000       0.5000000000
      0.2500000000       0.2500000000       0.2500000000
      0.7500000000       0.7500000000       0.2500000000
      0.2500000000       0.7500000000       0.7500000000
      0.7500000000       0.2500000000       0.7500000000
Total free energy (Ha): -3.133203267573469E+01
Atomic forces (Ha/Bohr):
 -1.7627851291E-02  -1.8267169314E-02  -2.0532228601E-02
 -3.2326283082E-03  -3.9157915997E-03  -8.4346817106E-03
 -2.1183397660E-03  -4.0727113668E-03  -4.7525385191E-03
 -3.1767964491E-03  -6.3409438358E-03  -4.5868359386E-03
  3.1114154829E-02   2.8991566680E-02   2.8418338450E-02
  1.8720663436E-04  -1.5325634374E-03  -1.5279713291E-03
 -9.2298364017E-03   9.7871710948E-03   9.4203499419E-03
  4.0840907529E-03  -4.6495582209E-03   1.9955677068E-03
Stress (GPa): 
 -9.4641683356E+00  -2.3845765509E+00  -2.0914427416E+00 
 -2.3845765509E+00  -9.5776637198E+00  -1.7064538404E+00 
 -2.0914427416E+00  -1.7064538404E+00  -9.5655265581E+00
