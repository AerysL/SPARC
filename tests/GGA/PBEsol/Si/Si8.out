***************************************************************************
*                           SPARC (Nov 06, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Dec 19 11:34:47 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 30 30 30
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999932524
CHEB_DEGREE: 23
NSTATES: 30
EXCHANGE_CORRELATION: GGA_PBEsol
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 25
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 5 5
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  25
Mesh spacing                       :  0.34 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  ./Si.psp8
Pseudocharge radii of atom type 1  :  8.16 8.16 8.16 (x, y, z dir)
Number of atoms of type 1          :  8
Fractional coordinates of atoms of type 1     :
      0.0100000000       0.0200000000       0.0250000000
      0.5000000000       0.5000000000       0.0000000000
      0.0000000000       0.5000000000       0.5000000000
      0.5000000000       0.0000000000       0.5000000000
      0.2500000000       0.2500000000       0.2500000000
      0.7500000000       0.7500000000       0.2500000000
      0.2500000000       0.7500000000       0.7500000000
      0.7500000000       0.2500000000       0.7500000000
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9112575086E+00        4.264E-02        0.974
2            -3.9152446324E+00        2.762E-02        0.233
3            -3.9161194581E+00        1.578E-02        0.235
4            -3.9164405321E+00        7.074E-03        0.236
5            -3.9164968123E+00        3.068E-03        0.234
6            -3.9165028676E+00        1.624E-03        0.231
7            -3.9165038568E+00        7.458E-04        0.235
8            -3.9165040147E+00        3.043E-04        0.235
9            -3.9165040594E+00        1.622E-04        0.227
10           -3.9165040869E+00        4.618E-05        0.225
11           -3.9165040875E+00        1.699E-05        0.221
12           -3.9165040873E+00        8.546E-06        0.221
13           -3.9165041003E+00        2.057E-06        0.222
14           -3.9165040683E+00        9.380E-07        0.210
Total number of SCF: 14    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9165040683E+00 (Ha/atom)
Total free energy                  : -3.1332032546E+01 (Ha)
Band structure energy              :  1.6535970228E+00 (Ha)
Exchange correlation energy        : -9.7361561714E+00 (Ha)
Self and correction energy         : -1.0219506067E+02 (Ha)
Entropy*kb*T                       : -4.3025069278E-04 (Ha)
Fermi level                        :  2.4127533393E-01 (Ha)
Atomic forces (Ha/Bohr):
 -1.7628187229E-02  -1.8267075449E-02  -2.0531885695E-02
 -3.2321505219E-03  -3.9159226761E-03  -8.4354625538E-03
 -2.1183059482E-03  -4.0731439401E-03  -4.7530880582E-03
 -3.1763831823E-03  -6.3415479646E-03  -4.5871044504E-03
  3.1114675975E-02   2.8992583257E-02   2.8419267369E-02
  1.8708661067E-04  -1.5325684068E-03  -1.5281437153E-03
 -9.2308078684E-03   9.7873967017E-03   9.4206831422E-03
  4.0840721642E-03  -4.6497215217E-03   1.9957339618E-03
RMS force                          :  1.6724930622E-02 (Ha/Bohr)
Maximum force                      :  5.1150246360E-02 (Ha/Bohr)
Time for force calculation         :  0.063 (sec)
Stress (GPa) : 
 -9.4642029350E+00  -2.3846285166E+00  -2.0914927357E+00 
 -2.3846285166E+00  -9.5776629929E+00  -1.7064376694E+00 
 -2.0914927357E+00  -1.7064376694E+00  -9.5655245112E+00 
Pressure                           :  9.5357968130E+00 (GPa)
Maximum stress                     :  9.5776629929E+00 (GPa)
Time for stress calculation        :  0.234 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  4.460 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
